.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,57 @@
+name: 'Storage dedupe/restore blobs task - nightly'
```

Comment:
Just call this "Nightly jobs"

This dedupe job is just one of them. Likely will add more later.